<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QDialog" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>546</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1098</width>
    <height>607</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="quit_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_button">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>180</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cube_roll">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>460</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Kubik</string>
   </property>
  </widget>
  <widget class="QWidget" name="kaznaWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>419</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
